js/ui/toolbar.js

```diff
@@ -169,30 +169,6 @@ var Toolbar = ToolbarBase.inherit({
                 options: {
                     submenuType: "actionSheet"
                 }
-            },
-            {
-                device: function() {
```

Comment:
@jsdmitry please review this part. Maybe the 'dropDownMenu', 'listBottom', 'listTop' submenuTypes now are useless.

Reply:
@jsdmitry @jauseg 
Guys, is this issue resolved?

Reply:
Yes, we have reviewed this submenuTypes. The 'dropDownMenu' and 'actionSheet' use in the code.

Reply:
BC?

Reply:
I think it does not BC. The submenuType is a private type. The value of this type depends on the theme. But these themes removed.